suites/pacific/rgw/tier-1-extn_rgw.yaml

```diff
@@ -68,7 +68,7 @@ tests:
   # STS tests
 
   - test:
-      name: STS Tests
+      name: STS Tests to Perform assume role on priciple user and perform IOs
```

Comment:
Can you please correct the typo error in "priciple"

Reply:
Okay Chaithra